README.md

```diff
@@ -2,7 +2,22 @@
 
 The Serial Monitor extension for Visual Studio Code provides a way to read from and write to serial ports.
 
-**This repository is used solely for tracking feedback. It contains no source code for the extension itself.**
```

Comment:
While this repo does contain source code now, I think we should still have some statement explaining that only the API lives here and the source code of Serial Monitor itself is still closed-source.

Reply:
Changing.